graphql_resolver.go

```diff
@@ -190,14 +196,16 @@ func (s *SessionResolver) PlayerSessions() ([]*PlayerSessionResolver, error) {
 
 	sr := make([]*PlayerSessionResolver, len(pSessions))
 
-	for i, s := range pSessions {
+	for i, session := range pSessions {
```

Comment:
https://github.com/golang/go/wiki/CodeReviewComments#variable-names

Reply:
Huh interesting, hadn't seen that wiki before. Good to know!